### ISSUE TITLE
Upgrade to Axe-core 4.0 when released

### ISSUE THREAD

**Issue by kruthivijay31** [storybookjs/storybook#10693]
**Is your feature request related to a problem? Please describe.**
We recently migrated to the storybook and found an a11y violation in the Switch component. 
<img width="455" alt="Screen Shot 2020-05-08 at 12 10 19 PM" src="https://user-images.githubusercontent.com/14045986/81441033-e335e900-9125-11ea-9955-c5b296effbd6.png">
We have raised this [issue with axe-core](https://github.com/dequelabs/axe-core/issues/2225) and I am assuming storybook is using axe-core for a11y validation.

**Describe the solution you'd like**
Upgrading to `Axe-core 4.0` will resolve the issue.

**Are you able to assist bring the feature to reality?**
I can with some guidance for storybook core team!

**Additional context**
If there is an alternative method to do this let us know! Thanks

**Comment by shilman** [comment]
Yay!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.1.0-alpha.1 containing PR #12150 that references this issue. Upgrade today to try it out!

You can find this prerelease on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.
